internal/provider/client_dashboard.go

```diff
@@ -6,11 +6,12 @@ import (
 	"strings"
 )
 
-func (c *SquaredUpClient) CreateDashboard(displayName string, workspaceId string, dashboardContent string) (*Dashboard, error) {
+func (c *SquaredUpClient) CreateDashboard(displayName string, workspaceId string, timeframe string, dashboardContent string) (*Dashboard, error) {
```

Comment:
How does adding a required argument (timeframe) impact upgrades of Terraform provider versions?  e.g. user upgrades to this version of provider and their Terraform deployment breaks as now they need to provide an extra string argument.

Reply:
`timeframe` is not a required argument. Its optional in `internal/provider/resource_dashboard.go`